Tests/test_PDB_ResidueDepth.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 by Francesco Gastaldello. All rights reserved.
+#
+# Converted by Francesco Gastaldello from an older unit test copyright 2002
+# by Thomas Hamelryck.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Unit tests for the Bio.PDB.ResidueDepth module."""
+
+import unittest
+from Bio.PDB import PDBParser, ResidueDepth
+
+
+class ResidueDepthTests(unittest.TestCase):
+    """Test ResidueDepth module."""
+
+    def test_ResidueDepth(self):
+        """Test on module that calculate residue depth via MSMS on protein structures."""
+        prot_file = 'PDB/2XHE.pdb'
+        p = PDBParser()
+        s = p.get_structure("X", prot_file)
+        model = s[0]
+        rd = ResidueDepth(model, prot_file)
+        res_chain = ''
+        for item in rd.property_list[:100]:
+            res_chain = res_chain + item[0].get_resname()
+        self.assertEqual(res_chain, """HISMETSERLEULYSSERALAVALLYSTHRVALLEUTHRASNSERLEUARGSERVALALAASPGLYGLYASPTRPLYSVALLEUVALVALASPLYSPROALALEUARGMETILESERGLUCYSALAARGMETSERGLUILELEUASPLEUGLYVALTHRVALVALGLUASPVALSERLYSGLNARGLYSVALLEUPROGLNPHEHISGLYVALTYRPHEILEGLUPROTHRGLUGLUASNLEUASPTYRVALILEARGASPPHEALAASPARGTHRPROTHRTYRGLUALAALAHISLEU""")
+        prot_file = 'PDB/2BEG.pdb'
```

Comment:
I would break the method here and start a new test method, e.g.

```python
    def test_ResidueDepth_ 2XHE(self):
        """Residue depth on 2BEG.pdb using MSMS."""
        prot_file = 'PDB/2XHE.pdb'
        ....

    def test_ResidueDepth_2BEG(self):
        """Residue depth on 2BEG.pdb using MSMS."""
        prot_file = 'PDB/2BEG.pdb'
        ....

    def test_ResidueDepth_ 1LCD(self):
        """Residue depth on 1LCD.pdb using MSMS."""
        prot_file = 'PDB/1LCD.pdb'
        ....

    def test_ResidueDepth_ 1A8O(self):
        """Residue depth on 1A8O.pdb using MSMS."""
        prot_file = 'PDB/1A8O.pdb'
        ....
```